src/palimpzest/query/optimizer/optimizer.py

```diff
@@ -16,14 +16,14 @@
     LogicalOperator,
     RetrieveScan,
 )
-from palimpzest.optimizer import (
+from ..optimizer import (
```

Comment:
I would like to eliminate any and all relative imports in favor of absolute imports if possible (these can mask circular import issues).

I.e. I would instead use:
```
from palimpzest.query.optimizer import (
    IMPLEMENTATION_RULES,
    TRANSFORMATION_RULES,
)
```